### PR TITLE
CircleCI构建修复

### PR DESCRIPTION
**Pull request by wangzul**
[CircleCI 构建参数bug修复]
[CircleCI 构建触发条件bug修复]

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -115,9 +115,10 @@ jobs:
 workflows:
   build-workflow:
     when:
-      or:
-        - equal: [ "publish_kuscia_deps", << pipeline.parameters.GHA_Meta >> ]
-        - equal: [ "publish_pypi", << pipeline.parameters.GHA_Meta >> ]
+      not:
+        or:
```
Comment (yushiqie) [inline .circleci/config.yml:119]: 这里为什么是or 而不是and

File: .circleci/deps-config.yml [hunk 2]
```diff
@@ -3,6 +3,18 @@
 version: 2.1
 
 parameters:
+  GHA_Actor:
```
Comment (yushiqie) [inline .circleci/deps-config.yml:6]: deps-config 都没用到这些参数，理论上不应该加

Comment (wangzul) [reply .circleci/deps-config.yml:6]: Previously, there was no prompt indicating that the parameters configuration was not configured. According to the test, not adding parameters will prompt unexpected parameters

File: .circleci/config.yml (hunk 1 above)
Comment (wangzul) [reply .circleci/config.yml:153]: According to the test results, the OR parameter operation logic is normal

File: .circleci/config.yml [hunk 3]
```diff
@@ -37,6 +37,34 @@ executors:
     shell: /bin/bash --login -eo pipefail
 
 commands:
+  test:
```
Comment (yushiqie) [inline .circleci/config.yml:40]: 这里的 test 没看到在哪里引用，看看是不是哪里拼写错误

**Review by yushiqie — APPROVED** [review]
lgtm

**Pull request merged**